### DISCUSSION TITLE
Tabs icon position

### DISCUSSION THREAD

**Discussion by aweell** [Telefonica/mistica-design#624] [Proposals]
Una propuesta que se ha comentado durante los últimos días es la posibilidad de cambiar la posición del icono entre:

* Izquierda de la label (actual)
* Superior de la label centrada.

Esta propuesta se plantea principalmente para tabs que se encuentran bajo el header y responden a una navegación primaria de la página.

![Screenshot 2022-07-22 at 10 55 26](https://user-images.githubusercontent.com/44420072/180403034-233c32c1-991a-422d-ba68-dc29953322bd.png)

**Comment by yceballost** [comment]
recuerdo haber tenido esta conversación, tenemos un caso de uso claro para el que sea necesario actualmente?

añado otra pregunta más..
Debería ser posible poder usar una tab con icono unicamente?

**Reply by aweell** [reply]
Para la primera pregunta voy a invocar a @HectorSancho, que fue con el que tuve la conversación.

A la segunda pregunta, valoraría si es realmente necesario plantear la opción sin texto, desde mi punto de vista es arriesgado y a no ser que hubiera una petición explícita, no lo incluiría en la definición del componente.

**Reply by yceballost** [reply]
> A la segunda pregunta, valoraría si es realmente necesario plantear la opción sin texto, desde mi punto de vista es arriesgado y a no ser que hubiera una petición explícita, no lo incluiría en la definición del componente.

agree.

Lo saco por el PreviewTools (0 relevante)
![image](https://user-images.githubusercontent.com/6722153/181014540-1ac5a9e0-4001-4cdd-885f-612344993b74.png)
